docs/source/tutorials/faq.ipynb

```diff
@@ -68,16 +68,26 @@
    "id": "3868ee8b",
    "metadata": {},
    "source": [
-    "### Why isn’t CleanLearning working for my data?"
+    "### Why isn’t CleanLearning working for me?"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "d13c9cd0",
    "metadata": {},
    "source": [
-    "At this time, CleanLearning only works directly with numpy matrices or pd.DataFrames.\n",
-    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities from your model via cross-validation and pass them as inputs."
+    "At this time, CleanLearning only works with data formatted as numpy matrices or pd.DataFrames, \n",
+    "and with models that are compatible with the `sklearn` API.\n",
+    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities (`pred_probs`) from your model via cross-validation and pass them as inputs.\n",
```

Comment:
link 'separately obtain predicted probabilities' to a tutorial where this is done or the the cleanlab.count.estimate_cv_predicted_probabilities here https://github.com/cleanlab/cleanlab/blob/master/cleanlab/count.py#L911

Reply:
added